app/containers/ReactionsList/AllTab.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Text, View, FlatList } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import Emoji from '../message/Emoji';
+import { useTheme } from '../../theme';
+import { IReaction, IApplicationState } from '../../definitions';
+import { TGetCustomEmoji } from '../../definitions/IEmoji';
+import I18n from '../../i18n';
+import styles from './styles';
+
+interface IAllReactionsListItemProps {
+	baseUrl: string;
+	getCustomEmoji: TGetCustomEmoji;
+	item: IReaction;
+	username: string;
+}
+
+interface IAllTabProps {
+	baseUrl: string;
+	getCustomEmoji: TGetCustomEmoji;
+	tabLabel: IReaction;
+	reactions?: IReaction[];
+	username: string;
+}
+
+const AllReactionsListItem = ({ item, baseUrl, getCustomEmoji, username }: IAllReactionsListItemProps) => {
+	const { colors } = useTheme();
+	const useRealName = useSelector((state: IApplicationState) => state.settings.UI_Use_Real_Name);
+	const count = item.usernames.length;
+
+	let displayNames;
+	if (useRealName && item.names) {
```

Comment:
I'm missing this fallback you did here https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4409/files#diff-e10dc1c26472d290df8dbc8947c55998127479dc23b34bb77c00c4d8e4505947R36

Reply:
Well isn't [this](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/fbd5cf414b281d069f385d9dada0456bfaeb7ed7/app/containers/ReactionsList/AllTab.tsx#L41) the same thing? I mean, `username` will be used if `useRealName`is false or the `name` is not available.

Reply:
Can you send me a screenshot?

Reply:
* When `useRealName` is false : 
<img width="360" alt="Screenshot 2022-09-02 at 2 41 47 AM" src="https://user-images.githubusercontent.com/77742477/188013978-4cf8de6c-9ac6-4b3c-8b73-d86e1924f4d4.png">

* When `useRealName` is true :
<img width="358" alt="Screenshot 2022-09-02 at 2 44 40 AM" src="https://user-images.githubusercontent.com/77742477/188014472-a960b04c-b38f-4445-952f-9bdfe55582d2.png">